techniques/aria/ARIA1.html

```diff
@@ -186,25 +181,25 @@ <h2>Related Resources</h2>
 
 
 <section id="related">
-      <h2>Related Techniques</h2>
+      <h2>関連テクニック</h2>
       <ul>
-         <li><a href="../aria/ARIA2">ARIA2: Identifying a required field with the aria-required property</a></li>
-         <li><a href="../aria/ARIA7">ARIA7: Using aria-labelledby for link purpose</a></li>
+         <li><a href="../aria/ARIA2">ARIA2: aria-required プロパティによって必須項目を特定する</a></li>
+         <li><a href="../aria/ARIA7">ARIA7: リンクの目的を示すために aria-labelledby を使用する</a></li>
      </ul>
    </section>
 <section id="tests">
-      <h2>Tests</h2>
+      <h2>検証</h2>
       <section class="procedure" id="procedure">
-         <h3>Procedure</h3>
+         <h3>手順</h3>
          <ol>
-            <li>Check that there is a user interface control having an <code class="language-html">aria-describedby</code> attribute that references one or more elements via unique <code class="language-html">id</code>.</li>
-            <li>Check that the referenced element or elements provide additional information about the user interface control.</li>
+            <li>一意な ID を経由して一つ以上の要素を参照する <code class="language-html">aria-describedby</code> 属性を持つユーザインタフェースコントロールがあることを確認する。</li>
+            <li>参照される要素 (一つ又は複数) が、ユーザインタフェースコントロールに関する追加情報を提供することを確認する。</li>
          </ol>
       </section>
       <section class="results" id="expected-results">
-         <h3>Expected Results</h3>
+         <h3>期待される結果</h3>
          <ul>
-            <li>Checks #1 and #2 are true.</li>
+            <li>#1 及び #2 の結果が真である。</li>
```

Comment:
「#1」というような日本語表記はふつうしないので、「#」は除去します。
```suggestion
            <li>チェック 1 及び 2 の結果が真である。</li>
```